covasim/webapp/cova_app.js

```diff
@@ -207,7 +207,7 @@ var vm = new Vue({
                 this.$set(this.int_pars, key, []);
             }
             // validate intervention
-            const notValid = !intervention.end || !intervention.start || intervention.end <= intervention.start
+            const notValid = !intervention.end || intervention.start < 0 || intervention.end <= intervention.start
```

Comment:
0 is a valid start day

Reply:
Is this documented anywhere?

Reply:
The sim start day is 0, so makes sense to allow interventions to start at the beginning. Actually we should allow the user to leave intervention.end blank as well (this is handled on the BE)